tests/unit/config/test_review_config.py

```
@@ -0,0 +1,140 @@
+    with pytest.raises(ValueError, match="review config must be a mapping"):
+        load_config(config_path=config_file)
+
+
+def test_load_config_ignores_unknown_checklist_keys(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """Unknown review.checklist keys warn instead of aborting config load."""
+    config_file = tmp_path / ".lintro-config.yaml"
+    config_file.write_text(
+        ("review:\n" "  checklist:\n" "    itemz:\n" "      - question: ignored\n"),
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)